lib/hyrax/configuration.rb

```diff
@@ -109,19 +109,41 @@ def register_roles
     attr_reader :analytics
     def analytics?
       @analytics ||=
-        ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYRAX_ANALYTICS', false))
+        ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYRAX_ANALYTICS', 'false'))
     end
 
-    attr_writer :google_analytics_id
-    def google_analytics_id
-      @google_analytics_id ||= nil
+    # Currently supports 'google' or 'matomo'
+    # google is default for backward compatability
+    attr_writer :analytics_provider
+    def analytics_provider
+      @analytics_provider ||=
+        ENV.fetch('HYRAX_ANALYTICS_PROVIDER', 'google')
+    end
+
+    # Date you wish to start collecting analytics for
```

Comment:
this is pretty minor, but it would be better if these were formatted with [yard](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md). that would make them readable with other documentation at https://www.rubydoc.info/gems/hyrax/Hyrax/Configuration

something like:

```suggestion
    ##
    # @!attribute [w] analytics_start_date
    #   @note this can be set using the +ANALITICS_START_DATE+ environment variable (format is YYYY-MM-DD)
    #   @return [String] date you wish to start collecting analytics for. used to compute the 
    #     "all-time" metrics.
```

Reply:
is it true that this is the date to start collecting analytics? or is it more like a filter?

Reply:
I guess its more like a filter.  Its the start_date part of the calculation for the all-time metrics.